internal/fetch/fetch.go

```diff
@@ -121,7 +126,12 @@ func latestSha(query string) (string, error) {
 
 // LatestCommitAndChecksum fetches the latest commit SHA and the SHA256 of the tarball for that
 // commit from the GitHub API for the given repository.
+// Defaults to [defaultBranchMaster] if the provided [Repo.Branch] is unset.
 func LatestCommitAndChecksum(endpoints *Endpoints, repo *Repo) (commit, sha256 string, err error) {
+	branch := defaultBranchMaster
```

Comment:
can we just pass in branch as an argument? it seems strange to have a default branch

Reply:
Ok, I think I've got all the necessary spots to "default" `repo.Branch` to `DefaultBranchMaster`.